data-avro/src/test/java/com/linkedin/data/avro/TestSchemaTranslator.java

```diff
@@ -322,6 +533,20 @@ public void testToAvroSchema_testTypeRefAnnotationPropagation(String schemaBefor
    assertEquals(fieldsPropertiesMap, avroSchemaAsDataMap);
  }
 
+  @Test(dataProvider = "toAvroSchemaData_testTypeRefAnnotationPropagation")
+  public void testToAvroSchema_testTypeRefAnnotationPropagation(String schemaBeforeTranslation,
```

Comment:
Can you please update the test method name to camelcase to follow the method naming style?
Same as data provider name.

Reply:
Will do

Reply:
It still has '_'

Reply:
corrected